train.py

```diff
@@ -198,7 +199,37 @@
   )
 
   def make_env():
-    return nmmo.Env(config)
+    import pickle as pkl
+    import numpy as np
+    import random
+
+    import os
+    print('cwd', os.getcwd())
+    with open('./pickled_task_with_embs.pkl', 'rb') as f:
```

Comment:
Put this file somewhere for us... might just have to be drive or w/e if it's large